Выволнен класс контроля над системой

Alex-Sergeev opened the pull request.

Alex-Sergeev committed:
Выволнен класс контроля системы

Alex-Sergeev committed:
Изменеие рандома

alvls commented:
Сменить кодировку в файлах на UTF8, иначе русский текст на github-е не читается.

Alex-Sergeev committed:
Исправление ошибки

Приоритет операций не увеличивался

Alex-Sergeev commented:
А как сменить?

alvls commented:
В Студии выбирает Файл->Сохранить как. В окне сохранения у кнопки Save есть выпадающий список. Там выбираете сохранить в кодировке.

Alex-Sergeev committed:
Сменил кодировку

Alex-Sergeev commented:
Поменял, спасибо.

Alex-Sergeev committed:
Исправления ошибок

Не увеличивалось время исполнения. Были перепутаны переменные время
исполнения, и текущее время исполнения

Alex-Sergeev committed:
Исправление опечаток в словах
